source/braille.py

```diff
@@ -21,9 +21,10 @@
 
 #: The directory in which liblouis braille tables are located.
 TABLES_DIR = r"louis\tables"
+PATTERNS_TABLE = os.path.join(TABLES_DIR, "braille-patterns.cti")

```

Comment:
nit: UNICODE_BRAILLE_TABLE or something might be a better name for this. IMO, braille-patterns.cti is a terrible name. This table allows Unicode braille characters to be used anywhere to produce raw dots.